cross/lame/Makefile

```diff
@@ -13,6 +13,29 @@ COMMENT  = High quality MPEG Audio Layer III encoder
 LICENSE  = LGPL
 
 GNU_CONFIGURE = 1
+CONFIGURE_ARGS += --with-Dll=no --with-debian=no
```

Comment:
These looked out of place, and with good reason as it turns out:
`config.log` shows `WARNING: unrecognized options: --with-Dll, --with-debian`.
Not sure why it was added, but it doesn't seem very useful to keep it around ;)